src/render/OpenGL.cpp

```diff
@@ -332,7 +332,14 @@ void CHyprOpenGLImpl::applyScreenShader(const std::string& path) {
     if (path == "" || path == STRVAL_EMPTY)
         return;
 
-    std::ifstream infile(absolutePath(path, std::string(getenv("HOME")) + "/.config/hypr"));
+    const char* xdgConfigHome = getenv("XDG_CONFIG_HOME");
```

Comment:
wouldn't it be better if we de-duplicated this?

we could make a CConfigManager::getConfigDir() and make CConfigManager::getMainConfigPath() depend on it, then just use getConfigDir() here